### DISCUSSION TITLE
Bug: Login with LDAP throws O_DIRECT not supported error

### DISCUSSION THREAD

**Discussion by marvinlanhenke** [minio/minio#20164] [Q&A]
## NOTE
I run minio in docker (FS-backend) with LDAP integration enabled.
Today, I switchted the setup from a named docker volume to a bind mount:

type cifs (rw,relatime,vers=3.0,cache=strict,username=xx,uid=0,noforceuid,gid=0,noforcegid,addr=xx,file_mode=0755,dir_mode=0755,soft,nounix,serverino,mapposix,rsize=4194304,wsize=4194304,bsize=1048576,echo_interval=60,actimeo=1,closetimeo=1)

## Expected Behavior

The LDAP login should work as before.

## Current Behavior

The LDAP login does not work: The following error is logged:
I checked and verified manually that O_DIRECT is supported.
I also tried to `mc admin config set minio api odirect='off'` with no success.

API: AssumeRoleWithLDAPIdentity
Time: 07:58:09 UTC 07/26/2024
DeploymentID: ffba2853-7717-4ece-85d4-3ae16bb17fa4
RequestID: 17E5B45001AD234F
RemoteHost: xx.xx.xxx.x
Host: 127.0.0.1:9000
UserAgent: Go-http-client/1.1
Error: drive does not support O_DIRECT (cmd.StorageErr)
       6: internal/logger/logger.go:268:logger.LogIf()
       5: cmd/logging.go:190:cmd.stsLogIf()
       4: cmd/sts-errors.go:43:cmd.writeSTSErrorResponse()
       3: cmd/sts-handlers.go:681:cmd.(*stsAPIHandlers).AssumeRoleWithLDAPIdentity()
       2: cmd/sts-handlers.go:359:cmd.(*stsAPIHandlers).AssumeRoleWithSSO()
       1: net/http/server.go:2166:http.HandlerFunc.ServeHTTP()

## Your Environment

minio version RELEASE.2024-05-10T01-41-38Z (commit-id=b5984027386ec1e55c504d27f42ef40a189cdb55)
Runtime: go1.22.3 linux/amd64
License: GNU AGPLv3 - https://www.gnu.org/licenses/agpl-3.0.html
Copyright: 2015-2024 MinIO, Inc.

**Comment by harshavardhana** [comment]
> I run minio in docker (FS-backend) with LDAP integration enabled.

what is the FS here?
